openrtb_ext/bid_response_video.go

```diff
@@ -14,7 +14,7 @@ type AdPod struct {
 }
 
 type VideoTargeting struct {
-	Hb_pb         string `json:"hb_pb"`
-	Hb_pb_cat_dur string `json:"hb_pb_cat_dur"`
-	Hb_cache_id   string `json:"hb_cache_id"`
+	HbPb       string `json:"hb_pb"`
+	HbPbCatDur string `json:"hb_pb_cat_dur"`
+	HbCacheId  string `json:"hb_cache_id"`
```

Comment:
In view of the earlier discussion of GoLint, why HbCacheId instead of HbCacheID?

Related to this, I see other "Id" instances in the above code (minimally, HbDealIdConstantKey).  Are they outside the scope of what's being addressed here?

Reply:
That should be `HbCacheID`. My bad! 

And about the "Id" instances, there are a lot of violations of those and other GoLint issues in our code which we should address. We will probably not be able to address all of them in one PR but I agree, refactoring those targeting keys in `openrtb_ext/bid.go` should be minimal and can probably be done as part of this PR. Thanks for the catch Joe. I will update the PR